### PR TITLE
Added Remaining Control Files and Texture Usage Guide

### PR DESCRIPTION
**Pull request by DevGeniusCode**
This pull request adds the remaining control files to the documentation, completing the reference for all control types. Additionally, a new guide has been included for working with textures in .wnd files. Key updates include:

1. **Completion of Control Files Documentation:**
   - Added documentation for all remaining control types in the .wnd file format.
   - Each control type is thoroughly described, including its attributes and usage.

2. **New Guide for Textures:**
   - A guide has been added to explain how to work with textures in .wnd files.
   - Includes instructions on how to define, configure, and apply textures to controls.
   - Details on texture file formats, dimensions, and how to link them to controls for visual customization.

These additions aim to provide a full understanding of control structures and visual customization through textures in the .wnd system.

### CODE REVIEW

File: documents/wnd/texturing.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                  IMAGE: NoImage, COLOR: 255 255 255 0, BORDERCOLOR: 255 255 255 0;
+```
+
+Here, `LoadingBar_R`, `LoadingBar_L` and other are the names of textures defined in the
+`SCSmShellUserInterface512.ini` file. The `NoImage` entries must still be included in their correct slots, even if there is no specific texture for that part of the control.
+
+**Important:** Each control has it's own way of setting up `DRAWDATA`, for example,  a `LoadingBar_Progress` must have the progress bar texture in slot 7, so if you try to use that slot for anything else, the progress bar will not be shown.
+
+It's important to use the example for each specific control and keep the textures on the exact slot, so the rendering will be correct, or you may cause the game to crash.
+## Creating and Modifying Textures
```
Comment (xezon) [inline documents/wnd/texturing.md:103]: \n

File: documents/wnd/texturing.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+3. **Test Your Changes:**
+    * Save both the edited TGA and INI files.
+    * Launch the game and check that the changes to the UI are applied correctly.
+
+### Important Notes
+
+* **Coordinate Precision**: Be accurate when recording the texture coordinates in the INI file.
+* **Texture Overlap**: Avoid having textures that overlap in the TGA file.
+* **Backup:** Always back up the original files before editing.
```
Comment (xezon) [inline documents/wnd/texturing.md:129]: This is not needed when using git or similar.

File: documents/wnd/texturing.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+* **TGA Format**: Save your images as a 32 bit uncompressed TGA file.
```
Comment (xezon) [inline documents/wnd/texturing.md:130]: In Patch development we use PSD or TIFF files. TGA saves no layers.

Also, it is possible to reference DDS DXT1, DXT5 in WND.

File: documents/wnd/controls/user.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
+# USER Control
+
+The **USER** control is the default control type, used for multiple purposes. It can serve as a container or group for other controls, as well as a tool for drawing rectangular shapes or lines by defining their position. Other controls also use the tags defined for the **USER** control, in addition to their own specific tags.
+
+## Purposes of the Control:
+1. **Container/Group**:
```
Comment (xezon) [inline documents/wnd/controls/user.md:6]: \n

**Review by xezon — APPROVED** [review]
Minor: Linebreaks behind titles "##" are sometimes inconsistent.

**Comment by DevGeniusCode** [comment]
Updated

**Comment by xezon** [comment]
github claims 2 files have conflicts:

documents/wnd/controls/combobox.md
documents/wnd/controls/user.md

**Comment by DevGeniusCode** [comment]
Updated

**Pull request merged**